assets/src/story-embed-block/block/index.js

```diff
@@ -18,9 +18,10 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-// TODO: Replace with web stories icon.
-import { page as icon } from '@wordpress/icons';
```

Comment:
Can you `npm remove @wordpress/icons` too?

Reply:
I was just thinking this as well. I need to check it isn't used anywhere else.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/2744/commits/5fc012f143d07c19823157d242867f3befaecb22